templates/index.html

```diff
@@ -6,7 +6,7 @@
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
           <div class="form-group mouse-over-text">
-            <label class="heading" for="bookmark">Fields with * marked after them are required<span>
+            <label class="heading" for="bookmark">Fields with * marked after them are required<br><span>
```

Comment:
add the `<br>`  after the ending `<span>` tag

Reply:
done